package-lock.json

```diff
@@ -5167,6 +5167,12 @@
           "resolved": "https://registry.npmjs.org/regexp-tree/-/regexp-tree-0.1.13.tgz",
           "integrity": "sha512-hwdV/GQY5F8ReLZWO+W1SRoN5YfpOKY6852+tBFcma72DKBIcHjPRIlIvQN35bCOljuAfP2G2iB0FC/w236mUw==",
           "dev": true
+        },
+        "semver": {
```

Comment:
Can you revert your `package-lock.json` changes? They aren't needed for the TypeScript migration.

Reply:
@devinbinnie I made the revert. Not sure why these changes occurred, any idea why?

Reply:
Whenever `npm install` is run, `package-lock.json` regenerates and moves/add/removes things. 
I'm not sure why to be honest. My personal strategy is to just not commit the file unless I'm updating/adding a specific package.

Reply:
That's a good one. Thinking about it now, I think I need to ditch my extravagant use of `git add .` :)